### PR TITLE
ISSUE_TEMPLATE: add templates for false negative and false positive tickets.

### PR DESCRIPTION
**Pull request by matthiaskrgr**
These will add the correct labels automatically so we don't have to do that ourselves :) 

changelog: none

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/false_negative.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+<code>
+```
+
+I expected to see this happen: *explanation*
+
+Instead, this happened: *explanation*
+
+### Meta
+
+- `cargo clippy -V`: e.g. clippy 0.0.212 (f455e46 2020-06-20)
```
Comment (flip1995) [inline .github/ISSUE_TEMPLATE/false_negative.md:25]: Unrelated thought: Should we remove the `0.0.212` part, when printing the Clippy version?

File: .github/ISSUE_TEMPLATE/false_positive.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+name: Bug Report (False Positive)
+about: Create a bug report about a lint warning that is wrong for Clippy
```
Comment (flip1995) [inline .github/ISSUE_TEMPLATE/false_positive.md:3]:
```suggestion
about: Create a bug report about a wrongly emitted lint
```

File: .github/ISSUE_TEMPLATE/false_negative.md (hunk 1 above)
Comment (matthiaskrgr) [reply .github/ISSUE_TEMPLATE/false_negative.md:25]: We could adopt a version scheme that follows rustc versions (similar to what cargo does).
nightly would be clippy 0.1.50, and stable clipy 0.1.49 then.

File: .github/ISSUE_TEMPLATE/false_positive.md (hunk 2 above)
Comment (flip1995) [inline .github/ISSUE_TEMPLATE/false_positive.md:3]: This hasn't changed. Rebase/squash gone wrong? :thinking:

Comment (matthiaskrgr) [reply .github/ISSUE_TEMPLATE/false_positive.md:3]: ah, I modified the false negative example accidentally ... :rofl:

File: .github/ISSUE_TEMPLATE/false_negative.md (hunk 1 above)
Comment (matthiaskrgr) [reply .github/ISSUE_TEMPLATE/false_negative.md:25]: created https://github.com/rust-lang/rust-clippy/issues/6499 for the version discussion

**Pull request merged**